mypy/build.py

```diff
@@ -2601,6 +2602,15 @@ def load_graph(sources: List[BuildSource], manager: BuildManager,
                 -1, -1,
                 "Duplicate module named '%s' (also at '%s')" % (st.id, graph[st.id].xpath)
             )
+            p1 = len(pathlib.PurePath(st.xpath).parents)  # type: int
+            p2 = len(pathlib.PurePath(graph[st.id].xpath).parents)  # type: int
```

Comment:
Do you really need these type comments? Mypy should be able to infer that `len()` gives an `int`.